pkg/component/etcd/service/values_helper.go

```diff
@@ -47,6 +47,7 @@ func GenerateValues(etcd *druidv1alpha1.Etcd) Values {
 		EtcdName:                 etcd.Name,
 		EtcdUID:                  etcd.UID,
 		Labels:                   etcd.Spec.Labels,
```

Comment:
do we need networking.gardener.cloud/* labels to be added to the service?

Reply:
I don't think that this PR is the place to address that. We need more context to rule out that label

Reply:
Then let us define the scope of the issue that this PR addresses. Since there is no hurry to get this PR out we can use this as an opportunity to clean up technical debt and streamline the labels that are added to all components that are created by druid.